builtin/logical/ssh/storage_migrations.go

```diff
@@ -0,0 +1,114 @@
+package ssh
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-uuid"
+	"github.com/openbao/openbao/sdk/v2/logical"
+)
+
+// This allows us to record the version of the migration code within the log entry
+const (
+	latestMigrationVersion = 1
+)
+
+type migrationInfo struct {
+	Created          time.Time `json:"created"`
+	MigrationVersion int       `json:"migrationVersion"`
+}
```

Comment:
Don't we still need to store the last-migrated legacy bundle and compare it to the current version in storage? If it is present and differs, it means we were downgraded and modified and thus need to rerun migration on upgrade.

Reply:
@Gabrielopesantos I guess, perhaps because issuer id is 1:1 to issuer key  material, IDs is sufficient as long as you never remove the key material as well.

Reply:
Ah, sorry, I think I was looking at an old version of this code. Resolving! Thank you!!